lib/improver/tests/utilities/__init__.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
```

Comment:
I'm not convinced that empty files need ownership information - there is nothing to own! What does @benfitzpatrick think?